packages/next/package.json

```diff
@@ -69,6 +70,7 @@
     "@next/polyfill-module": "11.0.2-canary.18",
     "@next/react-dev-overlay": "11.0.2-canary.18",
     "@next/react-refresh-utils": "11.0.2-canary.18",
+    "@node-rs/helper": "1.2.0",
```

Comment:
Should this be under `devDependencies` and compiled with ncc?

Reply:
It looks like this package isn't too big at [`82.8kB` installed](https://packagephobia.com/result?p=%40node-rs%2Fhelper) so I don't think it needs to be ncc'd but if it reduces the size some and doesn't affect the package it might help

Reply:
Reduced to [16.7kB installed](https://packagephobia.com/result?p=@node-rs/helper@1.2.1) in version `1.2.1`

Reply:
Since it's even smaller now and there appears to be an issue with ncc, I am going to keep it as is and mark this as resolved.